include/mbgl/util/projection.hpp

```diff
@@ -94,10 +94,11 @@ class Projection {
 
 private:
     static Point<double> project_(const LatLng& latLng, double worldSize) {
+        const double latitude = util::clamp(latLng.latitude(), -util::LATITUDE_MAX, util::LATITUDE_MAX);
         return Point<double> {
-            util::LONGITUDE_MAX + latLng.longitude(),
-            util::LONGITUDE_MAX - util::RAD2DEG * std::log(std::tan(M_PI / 4 + latLng.latitude() * M_PI / util::DEGREES_MAX))
-        } * worldSize / util::DEGREES_MAX;
+            180.0 + latLng.longitude(),
```

Comment:
@brunoabinader Is there a reason to remove the `util::LONGITUDE_MAX` and `util:DEGREES_MAX` constant in lieu of hard-coded values here?

Reply:
Good point - I replaced them while debugging to be able to understand better what was going on - will take those changes out :+1: